README.md

```diff
@@ -730,6 +744,11 @@ An order for placing the rule in `pg_hba.conf`. Defaults to `150`.
 ####`target`
 This provides the target for the rule, and is generally an internal only property. Use with caution.
 
+####`postgresql_version`
```

Comment:
It's already a given that it's postgresql, as it is attached to a define in the postgresql module. But it's your code, so I'll defer to your decision after having said my piece.